bin/elixir

```diff
@@ -132,7 +140,7 @@ while [ $I -le $LENGTH ]; do
         ;;
     --sname|--name)
         S=2
-        erl $(echo $1 | cut -c 2-)
+        erl "$(echo "$1" | cut -c 2-)"
```

Comment:
Good catch!

Reply:
yes, this is what shellcheck is really good at.